care/facility/models/asset.py

```diff
@@ -62,10 +64,8 @@ class Status(enum.Enum):
         choices=AssetClassChoices, default=None, null=True, blank=True, max_length=20
     )
     status = models.IntegerField(choices=StatusChoices, default=Status.ACTIVE.value)
-    current_location = models.ForeignKey(
-        AssetLocation, on_delete=models.PROTECT, null=False, blank=False
-    )
-    is_working = models.BooleanField(default=None, null=True, blank=True)
+    current_location = models.ForeignKey(AssetLocation, on_delete=models.PROTECT, null=False, blank=False)
+    is_working = models.CharField(choices=AssetWorkingChoices, default=None, null=True, blank=True, max_length=11)
```

Comment:
Change this back to the boolean field, if there are only two options then it does not make sense to not use boolean fields. also characters are not a logical value for a field named `is_working`

Reply:
done.